### PR TITLE
6 configuration

### PR DESCRIPTION
**Pull request by lchristus**

### CODE REVIEW

**Comment by lchristus** [comment]
im not sure what the current status of this branch is but it is required for the UI ticket so we need to have a look

File: src/MensaWidget/MWConfigurationTest.class.st [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+"
+Class {
+	#name : #MWConfigurationTest,
+	#superclass : #TestCase,
+	#category : #'MensaWidget-Tests',
+	#'squeak_changestamp' : 'Max 7/1/2024 20:26'
+}
+
+{
+	#category : #'as yet unclassified',
```
Comment (lchristus) [inline src/MensaWidget/MWConfigurationTest.class.st:15]: This method needs to be categorized

File: src/MensaWidget/MWConfigurationTest.class.st [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	#'squeak_changestamp' : 'Max 7/1/2024 20:21'
+}
+MWConfigurationTest >> testInvalidSet [
+	
+	MWConfiguration setPriceGroup: 'noPriceGroup'.
+	self assert: (MWMeal priceGroup isNil or: MWMeal priceGroup = 'students')
+]
+
+{
+	#category : #'as yet unclassified',
```
Comment (lchristus) [inline src/MensaWidget/MWConfigurationTest.class.st:25]: This aswell

File: src/MensaWidget/MWMeal.class.st
Comment (lchristus) [inline src/MensaWidget/MWMeal.class.st:1]: The class comment is outdated since the cars changed

**Review by lchristus — COMMENTED** [review]
I have no right to request changes since it is my PR officially but this needs to be changed otherwise it looks good but I didn’t test the tests or the functionality

File: src/MensaWidget/MWMeal.class.st
Comment (lchristus) [reply src/MensaWidget/MWMeal.class.st:1]: Vars*

**Review by lchristus — COMMENTED** [review]
I would approve if I could after testing functionality and the tests

File: src/MensaWidget/MWConfigurationTest.class.st [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	#'squeak_changestamp' : 'Max 7/1/2024 20:20'
+}
+MWConfigurationTest >> testValidSet [
+	
+	| oldPriceGroup |
+	
+	oldPriceGroup := MWMeal priceGroup.
+	MWConfiguration setPriceGroup: 'others'.
+	self assert: MWMeal priceGroup = 'others'.
+	MWMeal priceGroup: oldPriceGroup.
```
Comment (NilsKasper) [inline src/MensaWidget/MWConfigurationTest.class.st:35]: I thought the last line does not need a period after the statement

**Review by NilsKasper — APPROVED** [review]
Nice done

**Pull request merged**